_posts/2024-07-22-pyopensci-at-scipy-2024.md

```diff
@@ -0,0 +1,463 @@
+
+* Maintainers of packages that we have reviewed and accepted through our
+  [scientific Python software peer review process](/about-peer-review/index.html)
+* Colleagues who I have met at other meetings such as PyCon,
+* Reviewers and editors from our community,
+* Members of our advisory council, contributors, and
+* Friends—so many friends.
+
+This year, I was busy as SciPy:
+
+1. Running an in-person tutorial: Create Your First Python Package.
+2. Giving a talk in the Maintainers Track (what an honor!).
+3. Working the hallway track.
+4. Running a 1.5-day sprint that resulted in over 35 issues and pull requests.
+   Wow.
+
+Admittedly, I started off this conference behind and frazzled. You see, pyOpenSci
+has been growing in recent months. With that growth comes more wonderful people
+to support and engage with. More people getting involved does mean more work for
+pyOpenSci. However, the time is worth it, as all of this effort is moving
+pyOpenSci's mission of supporting open science forward.
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/july/scipy-24-pyopensci-package-submission-growth.png" alt="A bar plot with purple bars titled - number of submissions by quarter per year. The plot shows the number of submissions that pyOS has received since it started peer review in 2019. We started with only a few submissions per quarter but the numbers have grown to about 13 per quarter on average in the 2024. " style="width: 100%;" />
+  <figcaption><i class="fa-solid fa-heart-pulse"></i> pyOpenSci runs an open software peer review process. Above you can see how submissions have increased since we started in 2019. I started full time on pyOpenSci in late 2022. <i class="fa-solid fa-heart-pulse"></i>.</figcaption>
+</figure>
+
+In the end, even though I wasn't quite as prepared as I wanted to be, my talk
+and the workshop went great! This was a lesson for me in understanding that
+things can be less than perfect and still go well.
+
+More on all of that below. But first--what is the SciPy Conference?
+
+## About the SciPy Conference
+
+If you haven't attended this meeting before, let me give you the rundown. The
+SciPy Conference is an annual event dedicated to celebrating and learning more
+about scientific computing using the Python programming language. The meeting
+started back in [~2002](https://web.archive.org/web/20021030003450/http://www.scipy.org/site_content/scipy02/). The early meetings were run by EnThought and
+were smaller in size and scope. Over the years, SciPy has evolved significantly,
+mirroring the [🚀 explosive growth of the Python ecosystem 🚀](https://stackoverflow.blog/2023/01/26/comparing-tag-trends-with-our-most-loved-programming-languages/)
+in scientific computing and data science.
+
+Today, the SciPy Conference, hosted by [NumFocus](https://numfocus.org/),
+attracts over 700 attendees from diverse scientific fields such as astronomy,
+biology, geophysics, and more. SciPy is much more than just a workshop.
```

Comment:
```suggestion
biology, geophysics, and more. SciPy is much more than just a meeting.
```

workshop seems out of place, as so far in this section SciPy has been primarily referred to as a meeting.